reconciler/example/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
hm, not sure I'm super excited about a bash test case that doesn't run on `go test ./...`

What was the reasoning in not making this a Go test? ease of starting/stopping processes etc?